src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow.jsx

```diff
@@ -170,6 +170,14 @@ export default class SentryApplicationRow extends React.PureComponent {
     return this.props.installs && this.props.installs.length > 0;
   }
 
+  get installationStatus() {
+    if (this.props.installs && this.props.installs.length > 0) {
+      return capitalize(this.props.installs[0].status);
+    }
+
+    return 'Not Installed';
```

Comment:
Does this need a `t()`?

Reply:
This definitely should have a translation